src/NewTopicSuggestionForAdClusterForm.js

```diff
@@ -0,0 +1,115 @@
+import React, { useState } from 'react';
+import Form from 'react-bootstrap/Form';
+import Button from "react-bootstrap/Button";
+import Col from 'react-bootstrap/Col';
+import Row from 'react-bootstrap/Row';
+import Toast from 'react-bootstrap/Toast';
+import axios from "axios";
+
+const NewTopicSuggestionForAdClusterForm = params => {
+	var labelName = "";
+	const [validated, setValidated] = useState(false);
+	const [show, setShow] = useState(false);
```

Comment:
This controls whether the "success"/"failure" message is shown correct? please add this to the name so it's role is clear. something like showSubmitStatusMessage, or showSubmissionStatusMessage.

Reply:
Done